### PR TITLE
unit testing and aesthetic enhancements

### PR DESCRIPTION
**Pull request by ag4ml**

### CODE REVIEW

File: uvahelpr_models/marketplace/tests.py [hunk 1]
```diff
@@ -1,3 +1,126 @@
...
+        self.sample_user = User(username="faker", password="faker")
+        self.sample_user.save()
+        self.prof = Profile(phone_number="7324438712", skills="martial arts")
+        self.prof.user = self.sample_user
+        self.prof.save()
+
+    def test_getting_user_fails_bad_id(self):
+        request = self.factory.get('/api/v1/users/')
+        request.user = AnonymousUser()
+        response = UserRU.as_view()(request, 1*10**6)
```
Comment (mzwong) [inline uvahelpr_models/marketplace/tests.py:20]: What's with the 1_10_*6?

File: uvahelpr_models/marketplace/tests.py [hunk 2]
```diff
@@ -1,3 +1,126 @@
...
+        response = UserRU.as_view()(request, self.sample_user.id)
+        self.assertEqual(response.status_code, 200)
+        resp_dict = json.loads(response.content.decode())
+        self.assertTrue(resp_dict["ok"]==True)
+        self.assertTrue(len(resp_dict["result"])!=0)
+
+    def tearDown(self):
+         for user in User.objects.all():
+             user.delete()
+
```
Comment (mzwong) [inline uvahelpr_models/marketplace/tests.py:37]: What about testing update and delete for user?

File: uvahelpr_models/marketplace/tests.py [hunk 3]
```diff
@@ -1,3 +1,126 @@
...
+        result_dict = json.loads(response.content.decode())
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue(result_dict["ok"]==True)
+        self.assertTrue(len(result_dict["result"])!=0)
+
+    def tearDown(self):
+         for user in User.objects.all():
+             user.delete()
+         for message in Message.objects.all():
+             message.delete()
```
Comment (mzwong) [inline uvahelpr_models/marketplace/tests.py:126]: Same comment as above - should we be testing update and deletes?

**Review by mzwong — COMMENTED** [review]
Test cases look good for the most part. But are we missing update and delete tests?

File: uvahelpr_models/marketplace/tests.py (hunk 1 above)
Comment (ag4ml) [reply uvahelpr_models/marketplace/tests.py:20]: It's 1E6. Just an id that doesn't exist.

**Comment by ag4ml** [comment]
I talked to Leandra in my STS class and she said we only need to test the functionality related to the user stories that we implemented.

**Review by mzwong — APPROVED** [review]

**Review by will-gerard — APPROVED** [review]

**Pull request merged**